### PR TITLE
doc: Announcing the new hub

### PR DESCRIPTION
**Pull request by pilvikala**
This is the new Hub announcement. Text pending review/approval.

### CODE REVIEW

File: website/pages/blog/announcing-cloudquery-new-hub.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+<BlogHeader/>
+<iframe width="100%" height="480" src="https://www.youtube.com/embed/tJgzlZdXIPg"></iframe>
+
+We are excited to announce that we have launched [CloudQuery Hub](https://hub.cloudquery.io) - a new place to find, download, upload and buy CloudQuery plugins and add-ons (transformations & visualizations).
+
+## What Is the Hub?
+
+- The Hub provides dynamic documentation for every version of a plugin or add-on.
+- It enables fast downloads by hosting assets on dedicated infrastructure.
+- Plugins can now be published from any provider, such as GitLab or Gitea, not just GitHub, using [`cloudquery publish`](https://www.cloudquery.io/docs/reference/cli/cloudquery_plugin_publish) command.
```
Comment (disq) [inline website/pages/blog/announcing-cloudquery-new-hub.mdx:20]: Isn't the command now `cloudquery plugin publish`?

Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-new-hub.mdx:20]:
```suggestion
- Plugins can now be published from any provider, such as GitLab or Gitea, not just GitHub, using the [`cloudquery plugin publish`](https://www.cloudquery.io/docs/reference/cli/cloudquery_plugin_publish) command.
```

File: website/pages/blog/announcing-cloudquery-new-hub.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+To protect the CloudQuery Infrastructure that hosts plugins, we are going to introduce rate limiting per IP address when downloading plugins without logging in. If you frequently download plugins at a high rate, we recommend to either cache the downloaded assets or to log in to increase your rate limit.
+
+This change is not coming immediately. First, we are going to enable account creation and add support for authenticating and building production deployments. After that, for an initial one-month transition period, the rate-limiting will be disabled and plugins will be published to both CloudQuery Hub and GitHub. Thereafter the default rate limits will apply and new plugin versions will only be published to CloudQuery Hub.
+
+## Feedback & Future
+
+This is just the first release of the Hub, much more is coming. Stay tuned!
+
+We would love to hear what you think about the upcoming changes. If you have any questions, comments, or suggestions, please feel free to reach out to us on [Discord](https://cloudquery.io/discord) or [GitHub](https://github.com/cloudquery/plugin-sdk-python), or just [book a meeting](https://calendar.app.google/sAXWJWxfSEjUEsBAA).
```
Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-new-hub.mdx:32]:
```suggestion
We would love to hear what you think about the upcoming changes. If you have any questions, comments, or suggestions, please feel free to reach out to us on [Discord](https://cloudquery.io/discord) or [GitHub](https://github.com/cloudquery/cloudquery), or just [book a meeting](https://calendar.app.google/sAXWJWxfSEjUEsBAA).
```

File: website/pages/blog/announcing-cloudquery-new-hub.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+We are excited to announce that we have launched [CloudQuery Hub](https://hub.cloudquery.io) - a new place to find, download, upload and buy CloudQuery plugins and add-ons (transformations & visualizations).
+
+## What Is the Hub?
+
+- The Hub provides dynamic documentation for every version of a plugin or add-on.
+- It enables fast downloads by hosting assets on dedicated infrastructure.
+- Plugins can now be published from any provider, such as GitLab or Gitea, not just GitHub, using [`cloudquery publish`](https://www.cloudquery.io/docs/reference/cli/cloudquery_plugin_publish) command.
+
+## Other Important Notes
```
Comment (hermanschaaf) [inline website/pages/blog/announcing-cloudquery-new-hub.mdx:22]:
```suggestion
## Important Notes
```

File: website/pages/blog/announcing-cloudquery-new-hub.mdx (hunk 1 above)
Comment (erezrokah) [inline website/pages/blog/announcing-cloudquery-new-hub.mdx:20]:
```suggestion
- Plugins can now be published from any provider, such as GitLab or Gitea, not just GitHub, using [`cloudquery publish`](/docs/reference/cli/cloudquery_plugin_publish) command.
```

Let's also make this a relative link

**Review by yevgenypats — APPROVED** [review]
Nice!

**Pull request merged**